learning_journal/views/default.py

```diff
@@ -3,25 +3,56 @@
 
 from sqlalchemy.exc import DBAPIError
 
-from ..models import MyModel
+from ..models import Entry
 
 
-@view_config(route_name='home', renderer='../templates/mytemplate.jinja2')
+@view_config(route_name='list', renderer='../templates/list.jinja2')
 def my_view(request):
     try:
-        query = request.dbsession.query(MyModel)
-        one = query.filter(MyModel.name == 'one').first()
+        query = request.dbsession.query(Entry)
+        entries = query.all()
     except DBAPIError:
         return Response(db_err_msg, content_type='text/plain', status=500)
-    return {'one': one, 'project': 'learning_journal'}
+    return {'entries': entries, 'project': 'learning_journal'}
```

Comment:
Do you need to return a single key/value for 'project': 'learning_jounral' ?  Is this something you actually access in your journal and/or plan on changing?